File: EventFlowAPI/EventFlowAPI.Logic/DTO/Statistics/ResponseDto/ReservationStatistics.cs
namespace EventFlowAPI.Logic.DTO.Statistics.ResponseDto
{
    public class ReservationStatistics
    {
        public int AllNewReservationsCount { get; set; }
        public int AllCanceledReservationsCount { get; set; }
        public int AllNewEventReservationsCount { get; set; }
        public int AllNewFestivalReservationsCount { get; set; }
        public Dictionary<string, double> ReservationFestivalEventsDict { get; set; } = [];
        public Dictionary<string, double> ReservationTicketsTypesDict { get; set; } = [];
        public Dictionary<string, double> ReservationSeatTypesDict { get; set; } = [];

    }
}

File: EventFlowClient/src/models/response_models/stats/ReservationStatistics.ts
export type ReservationStatistics = {
  allNewReservationsCount: number;
  allCanceledReservationsCount: number;
  allNewEventReservationsCount: number;
  allNewFestivalReservationsCount: number;
  reservationFestivalEventsDict: Record<string, number>;
  reservationTicketsTypesDict: Record<string, number>;
  reservationSeatTypesDict: Record<string, number>;
};
